File: leet-code/Algorithms/001-two_sum/sol_a_brute_force.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans;
        for (int n = 0; n < nums.size(); n++)
        {
            for (int m = 0; m < nums.size(); m++)
            {
                if (nums[n] + nums[m] == target && m != n)
                {
                    ans.push_back(n);
                    ans.push_back(m);
                    return ans;
                }
            }
        }
        return ans;
    }
};

File: leet-code/Algorithms/1-two_sum/sol_a_brute_force.py
#!/bin/python3
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res = []

        for i in range(len(nums)):
            for j in range(len(nums)):
                if nums[i] + nums[j] == target and i != j:
                       res.append(i)
                       res.append(j)
                       return res

        return res
